packages/rest/test/acceptance/bootstrapping/application.acceptance.ts

```diff
@@ -45,10 +50,10 @@ describe('Bootstrapping the application', () => {
       class FooComponent {
         providers = {foo: FooProvider};
       }
-
       const app = new Application({components: [FooComponent]});
-
-      const value = app.getSync('foo');
+      const server = new RestServer(app);
+      app.server(server);
+      const value = server.getSync('foo');
```

Comment:
ditto: I think this change should be reverted - components are bound at application level now, right?